Cargo.toml

```diff
@@ -30,6 +30,12 @@ name = "dy"
 path = "src/main.rs"
 
 [dependencies]
+aws-config       = "1.4.0"
+aws-sdk-dynamodb = "1.28.0"
+aws-sdk-ec2      = "1.42.0"
+aws-smithy-runtime-api = "1.6.0"
+aws-smithy-types = "1.1.9"
+aws-types        = "1.2.1"
```

Comment:
We can remove this dependency because `aws_types::region::Region` is reexported as `aws_config::Region`.